core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -4015,6 +4015,34 @@ public void checkOrderBy(final boolean desc,
             "commission=null");
   }
 
+  /** Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-6948">[CALCITE-6948]
+   * Implement IntersectToSemiJoinRule</a>. */
+  @Test void testMinusToAntiJoinRule() {
```

Comment:
maybe you will be able to write a few quidem tests too after we merge   https://github.com/apache/calcite/pull/4301

Reply:
Yes, I am submitting #4301 to the draft branch now and would like everyone to see if this approach is acceptable. Then I will adjust the implementation method according to everyone's suggestions. The cases written in JdbcTest can all be completed using quidem tests.